pages/juang961225/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0px;
```

Comment:
Hey buddy I think you should use this general reset out of the body selector.

Reply:
solved